package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "eslint --fix ./src/*/*.ts",
     "format": "prettier -w .",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register 'tests/**/*.ts'"
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha -r ts-node/register --timeout 10000 'tests/**/*.ts'"
```

Comment:
hacky af :(

Reply:
Ahhh, had to do this because my hotspot was spotting :(. Questionable practice anyway to make real API calls like this but c'est la vie, this is the MVP